.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
   "editor.formatOnSave": true,
+  "prettier.trailingComma": "none",
```

Comment:
- may i know where is this being used for?

Reply:
Whenever I save sir, it automatically put a comma at the end of each array which is an error in Eslint. I already unchecked the format on save on my vs code settings but it is still occurring. So I decided to put this line of code in my settings.json instead.